GameData/KerbalismConfig/Support/ROCapsules.cfg

```diff
@@ -548,29 +937,40 @@
 		%max_pressure = 0.35
 	}
 }
+@PART[ROC-D2MissionModule2]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[RO-Kerbalism]
+{
+    @MODULE[Habitat]
+	{	
+		volume = 5.5        //guesstimate
+		surface = 4.347         //m^2
+		max_pressure = 0.35
+	}
+}
 @PART[ROC-ApolloCM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{	
-		%volume = 5.9
-		%surface = 15.71	 //guesstimate
+		%volume = 6.2	//m^3
+		%surface = 3.3749	 //m^2
 		%max_pressure = 0.35
 	}
 }
 @PART[ROC-ApolloCMBlockIII]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{
-		%volume = 6.2	//override default values to account for extra space with removed cargo racks
-		%surface = 15.71	 //guesstimate
+		%volume = 6.2	//m^3
+		%surface = 3.3749	 //m^2
+		%max_pressure = 0.35
 	}
 }
 @PART[ROC-ApolloMissionModule]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{
-		%volume = 16.8	//According to ETS wiki
-		%surface = 26.4	//guesstimate
+		%volume = 16.8	//m^3
+		%surface = 6.5595	//m^2
```

Comment:
Same as before. I also wouldn't remove the previous comments, they're useful enough. Repeating m^3 and m^2 over and over is redundant